### ISSUE TITLE
HBN fetcher failed

### ISSUE THREAD

**Issue by skoudoro** [dipy/dipy#2949]
## Description

During doc generation, I encounter this new issue:

```python
/Users/skoudoro/devel/dipy/doc/examples_revamped/reconstruction/reconst_fwdti.py failed leaving traceback:
Traceback (most recent call last):
  File "/Users/skoudoro/devel/dipy/doc/examples_revamped/reconstruction/reconst_fwdti.py", line 58, in <module>
    data_path = fetch_hbn(["NDARAA948VFH"])[1]
  File "/Users/skoudoro/devel/dipy/dipy/data/fetcher.py", line 2025, in fetch_hbn
    client.download_file("fcp-indi", download_files[k], k)
  File "/opt/homebrew/Caskroom/miniforge/base/envs/dipy-39-x86/lib/python3.9/site-packages/boto3/s3/inject.py", line 190, in download_file
    return transfer.download_file(
  File "/opt/homebrew/Caskroom/miniforge/base/envs/dipy-39-x86/lib/python3.9/site-packages/boto3/s3/transfer.py", line 326, in download_file
    future.result()
  File "/opt/homebrew/Caskroom/miniforge/base/envs/dipy-39-x86/lib/python3.9/site-packages/s3transfer/futures.py", line 103, in result
    return self._coordinator.result()
  File "/opt/homebrew/Caskroom/miniforge/base/envs/dipy-39-x86/lib/python3.9/site-packages/s3transfer/futures.py", line 266, in result
    raise self._exception
  File "/opt/homebrew/Caskroom/miniforge/base/envs/dipy-39-x86/lib/python3.9/site-packages/s3transfer/tasks.py", line 139, in __call__
    return self._execute_main(kwargs)
  File "/opt/homebrew/Caskroom/miniforge/base/envs/dipy-39-x86/lib/python3.9/site-packages/s3transfer/tasks.py", line 162, in _execute_main
    return_value = self._main(**kwargs)
  File "/opt/homebrew/Caskroom/miniforge/base/envs/dipy-39-x86/lib/python3.9/site-packages/s3transfer/download.py", line 642, in _main
    fileobj.seek(offset)
  File "/opt/homebrew/Caskroom/miniforge/base/envs/dipy-39-x86/lib/python3.9/site-packages/s3transfer/utils.py", line 378, in seek
    self._open_if_needed()
  File "/opt/homebrew/Caskroom/miniforge/base/envs/dipy-39-x86/lib/python3.9/site-packages/s3transfer/utils.py", line 361, in _open_if_needed
    self._fileobj = self._open_function(self._filename, self._mode)
  File "/opt/homebrew/Caskroom/miniforge/base/envs/dipy-39-x86/lib/python3.9/site-packages/s3transfer/utils.py", line 272, in open
    return open(filename, mode)
FileNotFoundError: [Errno 2] No such file or directory: '/Users/skoudoro/.dipy/HBN/derivatives/qsiprep/sub-NDARAA948VFH/anat/sub-NDARAA948VFH_desc-brain_mask.nii.gz.DDFCbDba'
```

it seems some path or files have changed in the server. Not sure how to result that. any idea @arokem or @36000 ? Thank you for your feedback

**Comment by arokem** [comment]
I can't reproduce this on my end (on current master). Is it possible that this is an intermittent error related to connectivity?

**Comment by skoudoro** [comment]
Maybe, let me try again, it was not the first time.

**Comment by skoudoro** [comment]
Ok, I confirm, it works. 

When it fails once because of connectivity, then it does not try to complete the missing data on a new run. You need to delete the HBN folder. 

Thanks !
